plugins/com.google.cloud.tools.eclipse.sdk.ui/src/com/google/cloud/tools/eclipse/sdk/ui/preferences/CloudSdkPreferenceArea.java

```diff
@@ -171,13 +173,33 @@ private void updateManagedSdk() {
     // would be nice if we could updateSelectedVersion() when finished
   }
 
-  private void updateSelectedVersion() {
+  // todo anyway to force this through the UI widgets rather than direct invocation?
```

Comment:
I don't understand this comment

Reply:
I think @elharo is talking about calling `area.updateSelectedVersion()` directly in the test (`testI2897()`). But in my testing, I didn't need to call it directly with correct changes.